package.json

```diff
@@ -14,66 +14,66 @@
     "start:dev": "npm run build:dev -- --watch"
   },
   "devDependencies": {
-    "@babel/core": "^7.8.7",
+    "@babel/core": "^7.9.0",
     "@babel/plugin-proposal-class-properties": "^7.8.3",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/plugin-transform-arrow-functions": "^7.8.3",
     "@babel/plugin-transform-regenerator": "^7.8.7",
     "@babel/plugin-transform-shorthand-properties": "^7.8.3",
     "@babel/polyfill": "^7.8.7",
-    "@babel/preset-env": "^7.8.7",
-    "@babel/preset-react": "^7.8.3",
-    "@babel/preset-typescript": "^7.8.3",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/preset-react": "^7.9.4",
+    "@babel/preset-typescript": "^7.9.0",
     "@centreon/frontend-core": "centreon/frontend-core",
-    "@svgr/webpack": "^5.2.0",
-    "@testing-library/jest-dom": "^5.1.1",
+    "@svgr/webpack": "^5.3.0",
+    "@testing-library/jest-dom": "^5.3.0",
     "@testing-library/react": "^10.0.1",
     "@testing-library/user-event": "^10.0.0",
     "@types/react-router": "^5.1.4",
-    "axios-mock-adapter": "^1.17.0",
-    "babel-jest": "^25.1.0",
-    "babel-loader": "^8.0.6",
+    "axios-mock-adapter": "^1.18.1",
+    "babel-jest": "^25.2.3",
+    "babel-loader": "^8.1.0",
     "babel-merge": "^3.0.0",
     "cache-loader": "^4.1.0",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^3.4.2",
     "eslint-config-airbnb": "^18.1.0",
-    "eslint-config-prettier": "^6.10.0",
+    "eslint-config-prettier": "^6.10.1",
     "eslint-import-resolver-alias": "1.1.2",
     "eslint-plugin-babel": "^5.3.0",
     "eslint-plugin-jest": "^23.8.2",
     "eslint-plugin-prefer-arrow-functions": "^3.0.1",
     "eslint-plugin-prettier": "^3.1.2",
     "file-loader": "^6.0.0",
-    "fork-ts-checker-webpack-plugin": "^4.1.0",
-    "html-webpack-plugin": "^3.2.0",
-    "jest": "^25.1.0",
+    "fork-ts-checker-webpack-plugin": "^4.1.2",
+    "html-webpack-plugin": "^4.0.2",
+    "jest": "^25.2.3",
     "jest-environment-jsdom-sixteen": "^1.0.3",
     "jest-junit": "^10.0.0",
     "lodash": "^4.17.15",
     "mini-css-extract-plugin": "^0.9.0",
     "node-sass": "^4.13.1",
     "postcss-loader": "^3.0.0",
     "postcss-safe-parser": "^4.0.2",
-    "prettier": "^1.19.1",
+    "prettier": "^2.0.2",
     "react-mock-router": "^1.0.15",
     "redux-mock-store": "^1.5.4",
     "resolve-url-loader": "^3.1.1",
     "sass-loader": "^8.0.2",
     "terser-webpack-plugin": "^2.3.5",
     "thread-loader": "^2.1.3",
-    "ts-loader": "^6.2.1",
+    "ts-loader": "^6.2.2",
     "typescript": "^3.8.3",
     "url-loader": "^4.0.0",
-    "webpack": "^4.42.0",
+    "webpack": "^4.42.1",
     "webpack-bundle-analyzer": "^3.6.1",
     "webpack-cli": "^3.3.11",
     "webpack-livereload-plugin": "^2.3.0",
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
     "@centreon/ui": "github:centreon/centreon-ui",
-    "@date-io/moment": "^1.3.13",
+    "@date-io/moment": "^2.5.0",
```

Comment:
this will break downtime dialog

Reply:
Why is that? can we adapt to the latest version?

Reply:
need to wait material-ui v4, which is currently in alpha: https://github.com/mui-org/material-ui-pickers/releases